meeting-notes/2018/2018-10-29--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,189 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 October 29, 2018
+
+- **Meeting Lead:** @eingenito
+- **Notetaker:**
+- **Attendees:**
+  - bigs
+  - erikingenito
+  - hannah howard
+  - warpfork
+  - djdv
+  - Adin Schmahmann
+  - Molly Mackinlay
+  - David Dias
+  - Kevin Atkinson
+  - Hector Sanjuan
+  - Travis Person
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+
+### Week Update
+
+@name
+ - Done:
+ - Blocked:
+ - Next:
+
+
+@marten-seemann
+
+- Done: completed the switch to tls-tris for QUIC (still a bunch of issues to fix though)
+- Blocked: nothing
+- Next: preparing for the IETF meeting next week
+
+@schomatis (not attending)
+- Done:
+  - Solved some issues and reviewed PRs on UnixFS and related components.
+  - Reviewed more UnixFS/MFS code.
+- Next: Coordinate with Rob to start producing some IPFS Files documentation.
+
+@warpfork
+ - Done:
+   - a few hours on refmt obj traversal improvements.
+   - researching prior art for IPLD schemas (protobuf, graphql, fuchsia FIDL) and gathering notes
+     - did you know?  basically nobody allows non-string map keys
+     - did you know?  graphql actually doesn't support maps at all (just structs)
+   - some experimental code for internal reprs of a type system.  just exploring.
+   - reading up on alternative chunkers.  need to digest more.  (ex; adler32 vs buzhash; casync switched from former to latter: why?)
+ - Blocked:
+   - more concrete IPLD data model specs is still on the wishlist.
+ - Next:
+   - need to do some PR reviewing (ex: ipld dagwalker)
+   - figure out where to land some of these notes and design sketches for IPLD
+  
+@travisperson
+ - Done:
+     - first pass at updating iptb
+ - Blocked: N / A
+ - Next:
+     - Mauppdate iptb with a few fixes
+     - update iptb to match ipfs gx deps
+     - release iptb 2.0
+  
+@aschmahmann
+ - Done:
+   - Basic ID-based message passing/protocol structure
+  - Research on GraphSync and planning next steps for graph synchronization related to multi-writer files
+ - Blocked:
+ - Next:
+  - Implement basic graph synchronization
```

Comment:
@aschmahmann in what level? Over IPFS or by implementing a GraphSync prototype?

Reply:
A graph synchronization prototype over libp2p where the graph nodes correspond to version changes (where the versions are stored in IPFS).